### DISCUSSION TITLE
Upgrade of GPS Speed in QRControl App

### DISCUSSION THREAD

**Discussion by breflick** [gopro/labs#420] [Feature Requests]
In addition to repeat feature, it would be nice to have these additional options in the QRControl as on the website:

 Disable auto power off

 Set LCD brightness to low (10%)

 Leave LCD on (Required on HERO10)

**Comment by dnewman-gpsw** [comment]
These are all in the Prefs tab on QRControl.  Under Auto Off, Back Light & LCD Off.

**Comment by breflick** [comment]
Thanks David.  So do I set those settings prior to setting the GPS speed control or is the sequence irrelevant?

Regards -- Brent

    On Tuesday, June 20, 2023 at 10:51:18 AM CDT, David Newman ***@***.***> wrote:  
 
 


These are all in the Prefs tab on QRControl. Under Auto Off, Back Light & LCD Off.

**Reply by dnewman-gpsw** [reply]
They are performed in the order listed in the command.

**Reply by breflick** [reply]
I meant -- do I scan the Prefs QR code before GPS Speed QR code when using the app or is the sequence of scanning these QR codes from app irrelevant?

**Reply by dnewman-gpsw** [reply]
Or scan them all at once. Try it.

**Comment by breflick** [comment]
Scanned Prefs QR code and GPS Speed settings appeared on GoPro screen.  Same result occurred following 2 additional repeat scans of Prefs QR code (for n=3).  Is this response supposed to be how GoPro responds to Prefs QR code?

**Reply by dnewman-gpsw** [reply]
What same result?  All QR Codes respond with a check mark, only some have a text message.

**Reply by breflick** [reply]
Same result meaning Prefs QR code scan resulted in GPS Speed setting appearing on the GoPro screen.

**Reply by breflick** [reply]
Check mark was observed following Prefs QR code scan prior to GPS Speed settings appearing on screen.

**Reply by breflick** [reply]
And to be clear, this issue is happening on a Hero11 Black.

**Reply by dnewman-gpsw** [reply]
You wrote "Leave LCD on (Required on HERO10)" ?

Both cameras?

The issue is not clear.  I have not idea what issue you are describing.

**Comment by breflick** [comment]
I have a Hero11 Black, only.  Apologies for not stating so to begin this discussion.  Here is the situation -- I open the QR iPhone app then select Prefs tab.  I enter parameters disabling auto power off, reducing LCD brightness and leaving LCD on.  The Prefs QR code is then scanned using the GoPro followed by observation of check mark then appearance of GPS Speed parameters on GoPro screen.

**Comment by breflick** [comment]
In prepping for potential use tonight, I went to QR iPhone app.  I noticed that the QR code is all-in-one.  I apologize for not noticing from the beginning but my initial anchor perspective was the website for the QR codes.  I think I got the app side for QR code generation now.  Thanks for your assistance and patience!
